R/fit.R

```diff
@@ -230,9 +230,9 @@ mmrm <- function(formula,
   nobs <- nrow(data)
   if (is.null(weights)){
     weights <- rep(1, nobs)
-  }
-  if(!is.null(weights) && any(weights <= 0)){
-    stop("only positive weights are allowed")
+    attr(weights, which = "dataname") <- "None"
```

Comment:
do we even need this then? because I see from the print code below that if the weights were not supplied by the user in the first place then it is not even printed at all

Reply:
current code prints `Weights:     None` when weights is not specified, otherwise we will have `Weights:         `.

Reply:
hm I am looking at it now, because currently we get 
`h_mmrm_tmb(formula = FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | 
    USUBJID), data = "fev_data", weights = "None", reml = reml, 
    control = control)`
as the `fit$call` result which is misleading